src/lsdb/nested/backends.py

```diff
@@ -0,0 +1,46 @@
+# Python 3.9 doesn't support "|" for types
```

Comment:
LSDB doesn't support python 3.9, though! No need to address in this PR - can be an easy follow-up.

Reply:
Will make an issue!